src/main/starlark/builtins_bzl/common/cc/cc_helper.bzl

```diff
@@ -1264,7 +1264,7 @@ cc_helper = struct(
     get_private_hdrs = _get_private_hdrs,
     get_public_hdrs = _get_public_hdrs,
     report_invalid_options = _report_invalid_options,
-    system_include_dirs = _system_include_dirs,
```

Comment:
can we keep system_include_dirs as well in cc_helper? It's used by some internal extension rules...

Reply:
put the old name back pointing to the same function